spec/base_spec.rb

```diff
@@ -16,6 +16,10 @@ def refresh_and_find_search_hit
     search_response.dig("hits", "hits").first
   end
 
+  def expect_response_models_match(response_models, expected_models)
```

Comment:
One thing to consider: This helper could also be responsible for asserting the model count, since every place this helper is used has `expect(response.models.count).to eq(n)` as well.

Reply:
I thought about just completely getting rid of the models count assertion..its redundant when we say that we expect the models to match and provide an array of elements